controller/src/beerocks/master/son_management.cpp

```diff
@@ -1714,6 +1714,39 @@ void son_management::handle_bml_message(Socket *sd,
         }
         break;
     }
+    case beerocks_message::ACTION_BML_MESSAGE_TO_RADIO_REQUEST: {
+        auto bml_request =
+            beerocks_header->addClass<beerocks_message::cACTION_BML_MESSAGE_TO_RADIO_REQUEST>();
+        if (bml_request == nullptr) {
+            LOG(ERROR) << "addClass CACTION_BML_MESSAGE_TO_RADIO_REQUEST failed";
+            break;
+        }
+        auto request = message_com::create_vs_message<
+            beerocks_message::cACTION_CONTROL_MESSAGE_TO_RADIO_REQUEST>(cmdu_tx);
+
+        if (request == nullptr) {
+            LOG(ERROR) << "Failed building CACTION_BML_MESSAGE_TO_RADIO_REQUEST message!";
+            break;
+        }
+
+        std::string dst_mac = tlvf::mac_to_string(request->radio_mac());
```

Comment:
you pulling the radio_mac from the wrong request 
```suggestion
        std::string dst_mac = tlvf::mac_to_string(bml_request->radio_mac());
```